CODEOWNERS

```diff
@@ -117,6 +118,13 @@ libs/ledger-live-common/src/hooks/recoverFeatureFlag.ts                 @ledgerh
 apps/ledger-live-desktop/tests/                          @ledgerhq/live-hub @ledgerhq/live-qa
 apps/ledger-live-mobile/e2e/                             @ledgerhq/live-hub @ledgerhq/live-qa
 
-# i18n files (no code owner)
+# Cross-team generic files can be opted out (no strict ownership when it concerns everyone)
 apps/ledger-live-desktop/static/i18n/
 apps/ledger-live-mobile/src/locales/
+apps/ledger-live-mobile/src/components/RootNavigator/
```

Comment:
I don't feel comfortable not having any owner on this part. Is there any way to break it down by files?

Reply:
That's one thing achieved with the new architecture: https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/newArch/features/Market/Navigator.tsx
Navigator composition